### DISCUSSION TITLE
Build without custom gears?

### DISCUSSION THREAD

**Discussion by VanDerLars** [FundamentalFrequency/.github#54] [Q&A]
Is it possible to build this awesome device without all the custom made elements and the Teensy? I mean, as an input, a regular keyboard could be enough. The only thing would be to translate the keyboard input into midi, doesnˋt it?

What do you think?
This would then be really affordable and easy to „build“.

Are there already ideas or solitions?

**Comment by VanDerLars** [comment]
Or, can we use another midi-board with enough keys and remap the inputs?

**Comment by stonepreston** [comment]
something like this is possible yes. I suppose you could write a keyboard listener program that just listens for keyboard events and then translates them to whatever midi you are wanting. Definitely make things more affordable, although loses much of the charm.
